quickjs.c

```diff
@@ -33742,6 +33742,30 @@ static JSValue js_global_isFinite(JSContext *ctx, JSValueConst this_val,
     return JS_NewBool(ctx, res);
 }
 
+static JSValue js_microtask_job(JSContext *ctx,
+                                int argc, JSValueConst *argv)
+{
+    JSValueConst func = argv[0];
+    JSValue ret = JS_Call(ctx, func, ctx->global_obj, 0, NULL);
+    if (JS_IsException(ret))
+        return ret;
+    JS_FreeValue(ctx, ret);
+    return JS_UNDEFINED;
+}
+
+static JSValue js_global_queueMicrotask(JSContext *ctx, JSValueConst this_val,
+                                        int argc, JSValueConst *argv)
+{
+    if (check_function(ctx, argv[0]))
+        return JS_EXCEPTION;
+
+    JSValueConst args[1];
+    args[0] = argv[0];
+    if (JS_EnqueueJob(ctx, js_microtask_job, 1, args))
```

Comment:
```suggestion
    if (JS_EnqueueJob(ctx, js_microtask_job, 1, &argv[0]))
```
Less convoluted IMO or is there a reason for writing it like that?

Reply:
I think the current compilation flags suppress the constness shenanigans so it shouldn't matter.